pages/event.tsx

```diff
@@ -0,0 +1,61 @@
+import { GetServerSideProps } from 'next'
+
+// components
+import Layout from '../components/Layout'
+import Navbar from '../components/Navbar'
+import Footer from '../components/Footer'
+import Container from '../components/Layout/Container'
+
+import picture from '../public/dummypost.png'
+
+// modules
+import HappeningNow from '../modules/EventPage/HappeningNow'
+import UpcomingSection from '../modules/EventPage/UpcomingSection'
+
+export interface PostProps {
+  userId: number
+  id: number
+  title: string
+  body: string
+}
```

Comment:
this is not a prop right, change the name to `IPost` or something

Reply:
Yep, But this interface uses in the Post component for the props. That's why I used the same interface to define server-rendered posts props and the Post Component props